inst/atime/tests.R

```diff
@@ -0,0 +1,69 @@
+pkg.edit.fun = function(old.Package, new.Package, sha, new.pkg.path) {
```

Comment:
Could you add pseudo-roxygen markup to describe the purpose of this function? It needn't be detailed, but a quick blurb about why we need it (I guess it's consumed by atime?) would be helpful for future readers

Reply:
this function is called for each version that we want to compare in the performance test (HEAD, master, historical regression, etc) and its job is to edit files so that we get an installable package with a name like data.table.abcdef1234567890 (SHA1 hash appended to package name) -- this is the mechanism that atime uses to test different versions of the code against each other, and this custom pkg.edit.fun is required because there are a few non-standard/repetitive places where "data.table" package name needs to be changed in the source files. This is documented in https://github.com/tdhock/atime/blob/main/vignettes/data.table.Rmd#L40 "It is not necessary to
specify `pkg.edit.fun` for typical packages (no compiled code, or
Rcpp), but `data.table` is an interesting example use case for
`pkg.edit.fun` since it specifies a custom shared object file name in
`Makevars`, and it has some custom version checking code in
`onLoad.R`."

Reply:
thanks, pointing to the right place in atime documentation would be sufficient for me